lib/classes/Variables.test.js

```diff
@@ -155,22 +155,21 @@ describe('Variables', () => {
       expect(newProperty).to.equal('my stage is dev');
     });
 
-    it('should populate non string variables', () => {
+    it('should populate number variables as sub string', () => {
```

Comment:
instead of replacing this test, could you please add another test for this case `should populate number variables as sub string` ... the test case you replaced is important to make sure if the variable is an array or object for example, it'll populate as expected